flow/project.py

```diff
@@ -1480,7 +1688,8 @@ def _add_dummy_operation(job):
         if profiling_results:
             print('\n' + '\n'.join(profiling_results), file=file)
 
-    def run_operations(self, operations=None, pretend=False, np=None, timeout=None, progress=False):
+    def run_operations(self, operations=None, pretend=False, np=None,
+                       timeout=None, progress=False):
```

Comment:
off-topic

Reply:
Artifact of rebasing. I had added some new argument that proved unnecessary originally.